plugin/input/file/worker.go

```diff
@@ -109,7 +110,15 @@ func (w *worker) work(controller inputer, jobProvider *jobProvider, readBufferSi
 						accumBuf = append(accumBuf, line...)
 						inBuf = accumBuf
 					}
-					job.lastEventSeq = controller.In(sourceID, sourceName, lastOffset+scanned, inBuf, isVirgin)
+
+					currOffset := lastOffset + scanned
```

Comment:
need more comments here)

Reply:
After discuss we chose different apporach.